modules/retail/pages/retail-deploy-terminals.adoc

```diff
@@ -120,7 +128,64 @@ image::retail_imagesync_formula.png[scaledwidth=80%]
 
 == Deploy Images to the Terminals
 
+When you have your bootstrap image ready, you can deploy the image to the terminals.
+
+
+
 .Procedure: Deploying Images to the Terminals
 
 . Power on your POS terminals.
 . The branch server will bootstrap the terminals according to the data you have provided.
+
+
+
+== Deploy Terminals with a Removable USB Device
+
+If you do not want to boot terminals from the network, you can create a live USB image and deploy terminals using a removable USB storage device.
+This is useful if you are unable or unwilling to connect some or all of your terminals to the network, or if you do not have a local {smr} branch server providing network services.
```

Comment:
This is not entirely connect. Terminals still need access to the network because the USB stick contains only boot image part usually served by PXE. The system image with OS and custom retail applications still needs to be fetched from SUMA server.

I think that use case for this feature are ATMs. They have network connection, but they do not have local branch which would provide PXE and tftp services. So they are booted using this USB stick (which may contain VPN configuration) and connect to some remote branch in the datacenter already through salt.